CONTRIBUTING.md

```diff
@@ -6,11 +6,11 @@ This document explains some of the guidelines around contributing to this projec
 
 ## Alignment first
 
-We ask you that if you have an idea for a new feature you [open an issue](https://git.musta.ch/francisco-diaz/SwiftInspector/issues/new) to discuss or reach out to [francisco-diaz](https://git.musta.ch/francisco-diaz) to align first.
+We ask you that if you have an idea for a new feature you [open an issue](https://github.com/fdiaz/SwiftInspector/issues/new).
```

Comment:
could this be
```suggestion
We ask you that if you have an idea for a new feature you [open an issue](issues/new).
```

Reply:
Maybe? Let's try it out.

Reply:
Works like this https://github.com/fdiaz/SwiftInspector/pull/34/files#diff-6a3371457528722a734f3c51d9238c13